plugin.yml

```diff
@@ -0,0 +1,14 @@
+name: ejson2env Buildkite Plugin
+description: Export environment variables stored & encrypted in EJSON files to the environment
+author: https://github.com/envato
+requirements: []
+configuration:
+  properties:
+    ejson_file:
+      type: string
+    ejson_private_key_env_key:
+      type: string
+  required:
+    - ejson_file
```

Comment:
Required even though it has a default value?

Reply:
Addressed in bde927d